src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -663,16 +665,20 @@ public Service(String reservedKey, String inId) {
     this.inbox = new Inbox(getFullName());
     this.outbox = new Outbox(this);
 
+    if (name.equals("i01")) {
```

Comment:
this is debugging code that probably shouldn't be included in this PR.

Reply:
sure, removed